### DISCUSSION TITLE
Use grapejs to generate Liquid templates

### DISCUSSION THREAD

**Discussion by mdwt** [GrapesJS/grapesjs#3617] [Q&A]
I would like to use grapejs to create Liquid templates, where the generated HTML contains the Liquid tags defined in my blocks, but the editor/canvas shows the actual rendered values.  

My blocks contain elements with Liquid syntax like
```
<p class="text-center">
          {{ cart.total }}
</p>
```

But the canvas would display it as
```
<p class="text-center">
          100
</p>
```

Is it possible to have the canvas render actual values, but the components being loaded/saved still contain the Liquid syntax?

**Comment by artf** [comment]
Yeah sure, you can create a custom component with a different view https://grapesjs.com/docs/modules/Components.html#view
